project/AppDependencies.scala

```diff
@@ -6,14 +6,15 @@ import sbt._
 object AppDependencies {
 
   private val bootstrapVersion = "8.1.0"
-  
+  val commonDomainVersion      = "0.10.0"
```

Comment:
Can this variable be inlined - looks like it will only be used once.

Reply:
It could, but its easier to have all of the api domain lib versions pulled up for ease of seeing them. Not much of an issue now as we have 4 deps.